src/mixins/itext_behavior.mixin.ts

```diff
@@ -461,37 +401,21 @@ export abstract class ITextBehaviorMixin<
     this._tick();
     this.fire('editing:entered');
     this._fireSelectionChanged();
-    if (!this.canvas) {
-      return this;
-    }
-    this.canvas.fire('text:editing:entered', { target: this });
-    this.initMouseMoveHandler();
-    this.canvas.requestRenderAll();
-    return this;
-  }
-
-  exitEditingOnOthers(canvas: Canvas) {
-    if (canvas._iTextInstances) {
-      canvas._iTextInstances.forEach((obj) => {
-        obj.selected = false;
-        if (obj.isEditing) {
-          obj.exitEditing();
-        }
-      });
+    if (this.canvas) {
+      this.canvas.fire('text:editing:entered', { target: this });
+      const disposer = this.canvas.on('mouse:move', this.mouseMoveHandler);
+      this._canvasMoveEventDisposer = () => {
+        disposer();
+        delete this._canvasMoveEventDisposer;
+      };
+      this.canvas.requestRenderAll();
     }
   }
 
-  /**
-   * Initializes "mousemove" event handler
-   */
-  initMouseMoveHandler() {
-    this.canvas.on('mouse:move', this.mouseMoveHandler);
-  }
-
   /**
    * @private
    */
-  mouseMoveHandler(options) {
+  protected mouseMoveHandler(options) {
     if (!this.__isMousedown || !this.isEditing) {
```

Comment:
i don't want to change more on this PR, but looks like that there is an opportunity for simplfication if we find a way to handle this either on the canvas or adding and removing this event handler every mousedown/mouseup rather than on editing entered/exit.

Defintely lower priority, but i point it out because following what was .__isMouseDown and why do we need to set it off on all instances every mouse up is something you forget every time and you have to rediscover

Reply:
seems to have worked...